### PR TITLE
Update documentation for the preCICE website

### LINKED ISSUE #31: Add documentation of the Micro Manager to precice.org

### PR DESCRIPTION
**Pull request by erikscheurer**
closes #31

This PR is in combination with https://github.com/precice/precice.github.io/pull/270 adding documentation on the micro manager to the preCICE website.
The documentation will be accessible as "tooling".

### CODE REVIEW

**Comment by erikscheurer** [comment]
How do I get Latex formulas to render correctly on the preCICE website? I want to have inline latex code, not using `\$$` since this renders the code in a new line. At the moment, the formulas in the table for the adaptivity are not correct.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
 
-A Manager tool to facilitate solving two-scale (macro-micro) coupled problems using the coupling library [preCICE](https://github.com/precice/precice).
+# The Micro Manager
 
-## Installing the Micro Manager
+The Micro Manager is a tool to facilitate solving two-scale (macro-micro) coupled problems using the coupling library [preCICE](https://www.precice.org/).
+
+## Installation
+
+The Micro Manager is a Python package that can be installed using `pip` or manually. Make sure [preCICE](installation-overview.html) is installed before installing the Micro Manager. The Micro Manager is tested with preCICE version [2.5.0](https://github.com/precice/precice/releases/tag/v2.5.0).
```
Comment (IshaanDesai) [inline README.md:15]:
```suggestion
The Micro Manager is a Python package that can be installed using `pip`. Make sure [preCICE](installation-overview.html) is installed before installing the Micro Manager. The Micro Manager is compatible with preCICE version [2.5.0](https://github.com/precice/precice/releases/tag/v2.5.0).
```

File: README.md [hunk 2]
```diff
@@ -12,7 +22,7 @@ It is recommended to install [micro-manager-precice from PyPI](https://pypi.org/
 pip install --user micro-manager-precice
 ```
 
-If the dependencies are not installed, then `pip` will attempt to install them for you. If you encounter problems in the direct installation, see the [dependencies section](https://github.com/precice/micro-manager#required-dependencies) below for links to installation procedures of all dependencies.
+Installing preCICE is mandatory, other dependencies will be installed by `pip` if they are not already installed. If you encounter problems in the direct installation, see the [dependencies section](#required-dependencies) below.
```
Comment (IshaanDesai) [inline README.md:25]:
```suggestion
Unless already installed, the dependencies will be installed by `pip` during the installation procedure. preCICE itself needs to be installed separately. If you encounter problems in the direct installation, see the [dependencies section](#required-dependencies) below.
```

File: README.md [hunk 3]
```diff
@@ -21,145 +31,157 @@ If the dependencies are not installed, then `pip` will attempt to install them f
 Ensure that the following dependencies are installed:
 
 * Python 3 or higher
-* [preCICE](https://github.com/precice/precice/wiki)
-* [pyprecice: Python language bindings for preCICE](https://github.com/precice/python-bindings)
+* [preCICE](installation-overview.html) Version [2.5.0](https://github.com/precice/precice/releases/tag/v2.5.0)
```
Comment (IshaanDesai) [inline README.md:34]:
```suggestion
* [preCICE](installation-overview.html) [v2.5.0](https://github.com/precice/precice/releases/tag/v2.5.0)
```

File: README.md [hunk 4]
```diff
@@ -21,145 +31,157 @@ If the dependencies are not installed, then `pip` will attempt to install them f
...
+* [pyprecice: Python language bindings for preCICE](installation-bindings-python.html)
 * [numpy](https://numpy.org/install/)
 * [mpi4py](https://mpi4py.readthedocs.io/en/stable/install.html)
 
 #### Build and install the Manager using pip
 
-After cloning this repository, go to the project directory `micro-manager` and run `pip3 install --user .`.
+After cloning this repository, go to the project directory `micro-manager` and run
```
Comment (IshaanDesai) [inline README.md:41]:
```suggestion
After cloning this repository, go to the directory `micro-manager/` and run
```

File: README.md [hunk 5]
```diff
@@ -21,145 +31,157 @@ If the dependencies are not installed, then `pip` will attempt to install them f
...
+
+```bash
+pip install --user .
+```
 
 #### Build and install the Manager using Python
 
-After cloning this repository, go to the project directory `micro-manager` and run `python setup.py install --user`.
+After cloning this repository, go to the project directory `micro-manager` and run
```
Comment (IshaanDesai) [inline README.md:49]:
```suggestion
After cloning this repository, go to the directory `micro-manager/` and run
```

File: README.md [hunk 6]
```diff
@@ -21,145 +31,157 @@ If the dependencies are not installed, then `pip` will attempt to install them f
...
+The Micro Manager facilitates two-scale coupling between one macro-scale and many micro-scale simulations. It creates instances of several micro simulations and couples them to one macro simulation, using preCICE.
 
-### Steps to convert micro simulation code to a callable library
+An existing micro simulation code written in Python needs to be converted into a library with a specific class name and specific function names. The next section describes the required library structure of the micro simulation code. On the other hand, the micro-problem is coupled to preCICE directly. The section [couple your code](couple-your-code-overview.html) of the preCICE documentation gives more details on coupling existing codes.
```
Comment (IshaanDesai) [inline README.md:59]:
```suggestion
An existing micro simulation code needs to be converted into a library with a specific class name which has functions with specific names. The next section describes the required library structure of the micro simulation code. On the other hand, the macro simulation code is coupled to preCICE directly. The section [couple your code](couple-your-code-overview.html) of the preCICE documentation gives more details on coupling existing codes.
```

**Comment by IshaanDesai** [comment]
In addition to restructuring the documentation, lets also split it into several document which are linked from one main page (the README). Right now the README is too long and hence difficult to read.

**Pull request merged**